### PR TITLE
Retirement spanish urls

### PR DESCRIPTION
**Pull request by wpears**
Depends on #8424

Switches the retirement app to the `/es/` prefix format.

Also adds the standard language picker to the about page for the retirement app

## Testing
- Load up http://localhost:8000/consumer-tools/retirement/before-you-claim/
- Run through the tool, everything should function as normal.
- Click through to the about page via the [Where do these numbers come from?](http://localhost:8000/consumer-tools/retirement/before-you-claim/about/)
- This should work (and the page should have a language picker
- Give the language picker a try
- Switch to Español back in the main app
- Everything should work.

### CODE REVIEW

File: cfgov/apache/conf.d/redirects.conf [hunk 1]
```diff
@@ -252,9 +252,13 @@ RedirectMatch permanent (.*)/realestateprofessionals(.*)  $1/real-estate-profess
...
 # Retirement redirects
-RedirectMatch permanent (?i)^/retirement/claiming-social-security/?$ /retirement/before-you-claim/
-RedirectMatch permanent (?i)^/jubilacion/?$  /retirement/before-you-claim/es/
-RedirectMatch permanent (?i)^/retirement/?$  /retirement/before-you-claim/
+RedirectMatch permanent (?i)^/retirement/claiming-social-security/?$ /consumer-tools/retirement/before-you-claim/
+RedirectMatch permanent (?i)^/jubilacion/?$ /es/herramientas-del-consumidor/jubilacion/antes-de-solicitar/
+RedirectMatch permanent (?i)^/retirement/?$  /consumer-tools/retirement/before-you-claim/
+
+# Redirect from /es/ suffix urls
+RedirectMatch permanent (?i)^/consumer-tools/financial-well-being/es/ /es/herramientas-del-consumidor/bienestar-financiero/
```
Comment (anselmbradford) [inline cfgov/apache/conf.d/redirects.conf:260]: Is this financial-well-being entry supposed to be here, or is it that this just hasn't been rebased with main?

Comment (wpears) [reply cfgov/apache/conf.d/redirects.conf:260]: Hah must have double pasted (check lines 263, 264)

Comment (wpears) [reply cfgov/apache/conf.d/redirects.conf:260]: Fixed in e291951d34

**Review by anselmbradford — APPROVED** [review]
LGTM 👍 

Do we care about the backend coverage failure?

**Comment by wpears** [comment]
>Do we care about the backend coverage failure?

That coverage error is kinda a phantom, Haven't added any new logic, really.

**Pull request merged**